scripts/python/phdf_diff.py

```diff
@@ -64,8 +66,120 @@ def addPath():
     #sys.path.insert(0,myPath+'/../vis/python')
     #sys.path.insert(0,myPath+'/vis/python')
 
-def compare(files, all=False, brief=True, quiet=False, one=False, tol=1.0e-12):
-    """ compares two hdf files """
+def compare_metadata( f0, f1, tol=1.0e-12):
+    """ compares metadata of two hdf files f0 and f1. Returns 0 if the files are equivalent.
+
+        Error codes:
+            10 : Times in hdf files differ
+            11 : Attribute names in Info of hdf files differ
+            12 : Values of attribute names in Info of hdf files differ
+            13 : Meta variables (Locations, VolumeLocations, LogicalLocations, Levels) differ
+    """
+    ERROR_TIME_DIFF=10
+    ERROR_INFO_ATTRS_DIFF=12
+    ERROR_INFO_VALUES_DIFF=13
+    ERROR_META_VARS_DIFF=14
```

Comment:
```suggestion
    ERROR_INFO_ATTRS_DIFF=11
    ERROR_INFO_VALUES_DIFF=12
    ERROR_META_VARS_DIFF=13
```

Probably a leftover from an earlier iteration, but the indices between doc and code don't match.

Reply:
Good catch, I just fixed this